pom.xml

```diff
@@ -207,6 +207,8 @@
     <module>context-object</module>
     <module>thread-local-storage</module>
     <module>optimistic-offline-lock</module>
+    <module>single-table-inheritance</module>
+    <module>crtp</module>
```

Comment:
crtp module should be there in `master` branch already

Reply:
sure, I will remove it.